Procfile

```diff
@@ -1,3 +1,3 @@
 release: POSTGRES_STATEMENT_TIMEOUT=600s bundle exec rails db:chatwoot_prepare
-web: bin/rails server -p $PORT -e $RAILS_ENV
-worker: bundle exec sidekiq -C config/sidekiq.yml
+web: bundle exec rails ip_lookup:setup && bin/rails server -p $PORT -e $RAILS_ENV
+worker: bundle exec rails ip_lookup:setup && bundle exec sidekiq -C config/sidekiq.yml
```

Comment:
We could trigger this from the initializer. It would ensure that it stays consistent across all installation environments with out updating the documentation.

Reply:
@sojan-official Can you please make the change if it is a small one?